console/src/components/Services/Data/TableModify/ModifyActions.js

```diff
@@ -128,6 +128,57 @@ const resetPrimaryKeys = () => ({
   type: RESET_PRIMARY_KEY,
 });
 
+export const removeCheckConstraint = constraintName => (dispatch, getState) => {
+  const isOk = window.confirm('Are you absolutely sure?');
+  if (!isOk) return;
+
+  const { currentTable: tableName, currentSchema } = getState().tables;
+
+  const tableSchema = getState().tables.allSchemas.find(s => {
```

Comment:
use `findTable` util fn and call the variable just `table`. Will be doing this change across the codebase sometime soon. `tableSchema` is confusing with table's postgres schema